docs/blockchainwallets.md

```diff
@@ -0,0 +1,95 @@
+
+If you don't have custody of your private keys, you're entrusting your money to others, which is risky. To solve this problem, a number of online wallets now let you take care of your keys all by yourself or with the help of other people (via multi-signatures). So, it's important to look at how each wallet uses technology before choosing the best one for you.
+
+### Desktop wallets
+
+As the name suggests, a desktop wallet is software that you download to your computer and use there. Desktop wallets give you full control over your keys and cash, unlike web-based wallets. When you create a new desktop wallet, a file on your computer is created and saved. Because this file has the private key information you need to access your wallet addresses, it should be encrypted with a personal password.
+
+If you encrypt your desktop wallet, you will be asked to enter your password every time you open the app so that it can read the saved file. If you lose this file or forget your password, you will most likely lose access to your money, so it's important to have a backup of your wallet.
+
+Keep the file in a safe place. You can also export the associated seed phrase or private key. If your computer breaks or is inaccessible for some reason, you will still be able to get to your money on other devices.
+
+In general, desktop wallets are thought to be safer than most web wallets, but you must make sure your computer is free of viruses and spyware before installing and using a cryptocurrency wallet.
+
+### Mobile wallets
+
+Mobile wallets work the same way desktop wallets do, but they are mostly made for smartphone apps. These are very helpful because you can use QR codes to send and receive cryptocurrency.
+
+Because of this, mobile wallets are the best way to make daily payments and transactions. This makes them a good way to spend Bitcoin, BNB, and other cryptocurrencies in the real world. One well-known example of a mobile crypto wallet is Trust Wallet.
+
+Like desktops, mobile devices can get messed up by rogue programs and malware. So, you should protect your mobile wallet with a password and make a copy of your private keys (or seed phrase) in case you lose or damage your phone.
+
+# The importance of backups
+
+It might be expensive to lose access to your cryptocurrency wallet. As a result, regular backups are essential. This is frequently accomplished by simply backing up wallet.dat files or seed phrases. A seed phrase, in essence, functions as a root key, generating and granting access to all keys and addresses in a crypto wallet. Also, if you chose password encryption, keep a backup of your password.
+
+# Supported wallets for the Effect Network

```

Comment:
I think this paragraph should be at the top, it's the most important bit for our users.